src/tests/ftest/telemetry/telemetry_pool_metrics.py

```diff
@@ -22,163 +20,155 @@ class TelemetryPoolMetrics(IorTestBase, TestWithTelemetry):
-                            # collecting total value for all targets
-                            temp = temp + value
-                total.append(temp)
-            # subtracting the total value after read/write from total value
-            # before read/write
-            final_value.append(total[1] - total[0])
-            # performing verification now
-            if "xferred" in name:
-                # check if oclass has any replication and get the replication
-                # numeric value, else use 1.
-                if True in [char.isdigit() for char in
-                            self.ior_cmd.dfs_oclass.value]:
-                    replication = re.search(
-                        r'\d+', self.ior_cmd.dfs_oclass.value).group()
-                else:
-                    replication = 1
-                # multiply the written amount by replication number
-                # to get total expected written amount for only
-                # write operation.
-                if "update" in name:
-                    expected_total_amount_written = (self.ior_cmd.block_size.value *
-                                                     int(replication))
-                else:
-                    expected_total_amount_written = self.ior_cmd.block_size.value
-                # get difference between actual written value and expected
-                # total written value.
-                final_value[-1] = final_value[-1] - expected_total_amount_written
-                # check whether the difference obtained in previous step is
-                # less thank the chunk size as expected_values[1] carries the
-                # chunk size value.
-                if final_value[-1] > expected_values[1]:
-                    self.fail("Aggregated Value for {} of all the targets "
-                              "is greater than expected value of {}"
-                              .format(final_value[-1], expected_values[1]))
-            else:
-                if not self.check_range(expected_values[0],
-                                        final_value[-1],
-                                        self.threshold_percent):
-                    self.fail("Aggregated Value for {} of all the targets "
-                              "is out of expected {} percent range"
-                              .format(final_value[-1], self.threshold_percent))
+        self.dfs_oclass = None
+        self.ior_transfer_size = None
+        self.ior_block_size = None
+
+    def setUp(self):
+        """Set up each test case."""
+        # Start the servers and agents
+        super().setUp()
+
+        self.dfs_oclass = self.params.get("oclass", '/run/container/*', self.dfs_oclass)
+        self.ior_transfer_size = self.params.get(
+            "transfer_size", '/run/ior/*', self.ior_transfer_size)
+        self.ior_block_size = self.params.get("block_size", '/run/ior/*', self.ior_block_size)
+
+    def get_expected_values(self, resent_ops):
```

Comment:
It looks like the inner dictionary contains min, max values, so you might want to explain that in the comment. You could also update the method name to
```suggestion
    def get_expected_value_range(self, resent_ops):
```

Reply:
- [ ] Add comments describing expected values table
- [ ] Update function names